index.html

```diff
@@ -105,7 +120,7 @@ <h3>Timeline</h3>
 	    <div class="eight columns">
             <h1>Travel</h1>
 		      <p>
-                    We don't want financial cost to be a barrier in attending TotalityHacks. If you're skilled, we want you. As such, we'll do our best in providing travel reimbursement. Along with your decision, you'll also receive a travel reimbursement cap.
+                    We don't want financial cost to be a barrier in attending TotalityHacks. If you're skilled, we want you. As such, we'll do our best in providing travel reimbursements. Along with your decision, you'll also receive a travel reimbursement cap.
```

Comment:
I actually prefer the singular reimbursement here because each person only gets one reimbursement, but maybe I'm wrong. @gmittal Thoughts?

Reply:
I didn't make the change yet because @gmittal may be a good second opinion